_data/projects/Instalution.yml

```diff
@@ -0,0 +1,12 @@
+name: Instalution
+desc: Instalution aims to be a revolutionary solution to Instagram.
+site: https://github.com/instalution/frontend
+tags:
+  - artist
+  - media
+  - portfolio
+  - frontend
+  - developer-network
+upforgrabs:
+  name: instalution
+  link: https://github.com/instalution/frontend/blob/master/README-en.md#short-list-of-needs--todos
```

Comment:
@groovenectar This should be the link to that LABEL. Eg: https://github.com/instalution/frontend/labels/good%20first%20issue